src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java

```diff
@@ -368,11 +368,6 @@ public void validateForStudentExamGeneration(Exam exam) throws BadRequestAlertEx
             }
         }
 
-        // Check that the exam max points is set
-        if (exam.getMaxPoints() == null) {
-            throw new BadRequestAlertException("The exam max points is not set.", "Exam", "artemisApp.exam.validation.maxPointsNotSet");
-        }
-
```

Comment:
Since now `exam.getMaxPoints` returns `int`, this check is redundant.

Reply:
I think max points cannot be 0, so you should check that the value is not 0 now